app/controllers/profiles_controller.rb

```diff
@@ -102,7 +102,7 @@ def render_footer?
 
   def typeahead
     suggester_options = []
-    suggestions = Profile.typeahead(params[:q])['suggest']
+    suggestions = Profile.typeahead(params[:q], current_region)['suggest']
```

Comment:
Let's quickly talk through this. Where does current region come from? What if it is not set?